prometheus/prometheus.yml

```diff
@@ -0,0 +1,18 @@
+scrape_configs:
+  - job_name: 'relay-miner'
+    scheme: http
+    static_configs:
+      - targets: ['relayminer-example:9090']
+  - job_name: 'appgate-server'
```

Comment:
Non-actionable. Just thinking outloud that we might want to rename `appgate server` to just `appgate` one day 😰